execution/executor-types/src/executed_chunk.rs

```diff
@@ -95,7 +97,11 @@ impl ExecutedChunk {
                 ..
             } = txn_to_commit;
             committed_transactions.push(transaction);
-            committed_events.extend(events);
+            committed_events.extend(
```

Comment:
nit: rename `committed_events` to `subscribable_events`?

Reply:
fixed